frontend/src/components/modal/job/JobDetailModal.vue

```diff
@@ -185,7 +193,11 @@
             id="job-save-button"
             color="blue-darken-1"
             variant="text"
-            @click="this.saveClicked"
+            @click="
```

Comment:
It looks like job creation is broken on this branch - nothing happens when I click save when trying to add a job 🤔

Reply:
Simply forgot to add `()` when adding more to the `@click` effect. Resolved.